killerbee/dev_bumblebee.py

```diff
@@ -17,17 +17,9 @@
 from datetime import datetime # type: ignore
 from .kbutils import KBCapabilities, makeFCS, bytearray_to_bytes # type: ignore
 
-# Import USB support depending on version of pyUSB
-try:
-    import usb.core # type: ignore
-    import usb.util # type: ignore
-    import sys # type: ignore
-    print("Warning: You are using pyUSB 1.x, support is in beta.", file=sys.stderr)
-except ImportError:
-    import usb # type: ignore
-    print("Error: You are using pyUSB 0.x, not supported for CC253x.", file=sys.stderr)
```

Comment:
Do we still need any warning about old pyUSB for CC253x series?

Reply:
No, support for 0.x pysub is being removed

Reply:
Great. Should we just have it do an error then and totally exit if in that state so people don't try it and then ask why?

Reply:
Actually, as long as setup.py has the version required specified that seems fair to just have the requirements explicitly document it.

Reply:
Updated setup.py to fail on usb 0.x and explicitly instruct to upgrade to 1.x